mx.py

```diff
@@ -1794,6 +1803,9 @@ def build(self, sourceFiles, project, jdk, compliance, outputDir, classPath, pro
         disableApiRestrictions, warningsAsErrors, showTasks):
         nyi('build', self)
 
+    def prepare(self, jdk, state):
```

Comment:
What is this for? In particular, when is it called? Please add some documentation.